SketchUp/Geom/BoundingBox.rb

```diff
@@ -253,10 +249,10 @@ def initialize
   #   boundingboxes actually overlap - then call this to get the resulting
   #   boundingbox.
   #
-  # @param boundingbox
+  # @param boundingbox [Geom::BoundingBox]
```

Comment:
We stick with `@param [types] name` - since that's what YARD will output when we generate the stubs. Sticking with path of least resistance as customizing YARD is a snowballing task.

(I didn't notice that comment had been merged into the guide. My bad.)